uk_bin_collection/uk_bin_collection/councils/LondonBoroughHammersmithandFulham.py

```
@@ -0,0 +1,57 @@
+# import the wonderful Beautiful Soup and the URL grabber
+class CouncilClass(AbstractGetBinDataClass):
+    """
+    Concrete classes have to implement all abstract operations of the
+    base class. They can also override some operations with a default
+    implementation.
+    """
+
+    def parse_data(self, page: str, **kwargs) -> dict:
+
+        user_postcode = kwargs.get("postcode")
+        check_uprn(user_postcode)
```

Ruff (0.14.14)

[error] `check_uprn` may be undefined, or defined from star imports

(F405)